clang_tools/install.py

```diff
@@ -54,9 +54,9 @@ def is_installed(tool_name: str, version: str) -> Optional[Path]:
         return None  # failed to locate the binary
     path = Path(path).resolve()
     print("at", str(path))
+    ver_num = ver_num.groups(0)[0].decode(encoding="utf-8").split(".")  # pragma: no cover
```

Comment:
Why `# pragma: no cover`?

Reply:
I'm surprised that it wasn't getting covered. From a glance, it should be covered if the binary is located.

I'm ok with keeping the line not covered as it would be a reminder that we should be updating the unit tests or writing a new one that triggers it.